text/0000-qa-team.md

```diff
@@ -0,0 +1,102 @@
+
+---
+
+# Summary
+[summary]: #summary
+
+* The QA team for Grin:
+  * works to ensure that a high standard of quality for contributions is upheld across the project
+  * develops tools and processes to help monitor and enforce the quality of work
+  * exists to make sure that changes to the project are quality, not to produce the quality changes themselves
+  * ensures that contributors are encouraged, supported and empowered to produce the best quality work to be integrated into Grin
+
+# Motivation
+[motivation]: #motivation
+
+Over time, with many people contributing to many different areas of Grin, some things can get lost in the mix including quality. It could be beneficial to have a team supported by members, processes and tools to help provide a consistent level of quality contributions.
+
+* Ensure that contributions to Grin are high quality
+* Empower contributors to make high quality contributions to Grin
+* Provide feedback from a QA perspective to micro and macro changes in Grin
+
+# Community-level explanation
+[community-level-explanation]: #community-level-explanation
+
+The QA team observes micro and macro changes occurring over time with various contributions to Grin. They help review PRs, have opinions on both development and governance decisions and provide tooling frameworks for contributors to use and maintain to ensure a consistent high level of quality of work across the Grin project. The QA team provides support for contributors to encourage them to continue to submit quality work.
+
+The day to day impact would be more attention for contributions and contributors. The long term impact would be more consistency in the level of quality work that makes it into the Grin project over time.
+
+## Example
+
+In this example a large PR is submitted that makes a nontrivial change to the Grin node.
+* Before submitting the PR, the contributor reads and integrates the well-formed contribution documentation to assist and support their efforts
+* Before the PR is accepted the contributor may use the Grin testing harness framework provided by the QA team.
+* Members of the QA team may manually review to changes for quality at a micro level: does the code do a good job of implementing the feature, is it readable, are there any obvious security issues, does it follow our contribution guidelines for formatting etc.
+* The QA may also review the change for quality at a macro level: are the changes in line with the other pieces of the project, does it add inconsistencies when interacting with other parts, does it advance the expected overall quality of the project etc.
+* Any necessary feedback is shared with the contributor to empower them to continue to make high quality contributions
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+## Membership
+
+* Membership is bootstrapped by @lehnberg
+* Minimum three, maximum seven members
```

Comment:
This feels like a very arbitrary requirement to me. It's not a security team, why do we need to have a cap and a floor?

Reply:
Agreed and removed